src/teuthology_api/models/presets.py

```diff
@@ -0,0 +1,11 @@
+from sqlmodel import Field, SQLModel, UniqueConstraint
+
+
+class Presets(SQLModel, table=True):
+    id: int = Field(primary_key=True)
+    username: str = Field(index=True)
+    name: str
+    suite: str
```

Comment:
We can discard the suite field since there are no benefits of having a suites field in the db unless we are trying to sort something by suite. @Devansh3712  what do you think?

Reply:
@kamoltat Devansh and I were discussing on slack about use-cases where suite as a field could help. I was thinking we could keep `suite` so users can explore different presets that other users have used for that suite then it would be helpful to query. Devansh came up with a great idea that we can also add an endpoint for getting presets by suites with pagination. 
What do you think about this?

Reply:
@kamoltat one of an interesting use case of having `suite` field in db would be that user can query their xyz suite presets. 
If I see on /runs page that you often run the rados suite, and I want to run rados too. Then I can essentially query presets under your username and rados suite. I can dry-run and see how many jobs that schedules. And play around with that.